### PR TITLE
Standardize node version

### PR DESCRIPTION
**Pull request by brunobar79**
We were using different versions in CI vs .nvmrc and also we were running an outdated version of node.

- Bump node version on CircleCI docker img from v8 to v8.11.3
- Bump .nvmrc node version from v6.3.1 to v8.11.3
- Update package-lock.json after running npm install with new node version

Looks like there's some improvements in the package-lock.json and they got rid of ~ prefix (that explains the huge diff) which in my opinion is a great call and will lead to more stable builds ...

### CODE REVIEW

**Comment by TrejGun** [comment]
Hi there!
it's possible to bump node to 10, in this case you need to add sha1 v 1.2.2 to deps. would you like to see such PR?

File: .nvmrc [hunk 1]
```diff
@@ -1 +1,2 @@
-v6.3.1
+v8.11.3
+
```
Comment (whymarrh) [inline .nvmrc]: Nit: this extra line can be removed

**Review by whymarrh — COMMENTED** [review]
This LGTM, though I might suggest closing #4568 in favour of this or vice-versa, but I'll leave that to @kumavis. Thanks for fixing this.

**Comment by brunobar79** [comment]
@whymarrh  Thanks for reviewing!  (I wasn't aware of #4568)

I think both PRs are addressing different things (node vs npm). If you and @kumavis agree I could upgrade to npm 6 and regenerate the package-lock.json on this PR and then we could def. close #4568

**Comment by kumavis** [comment]
closed via #4675

**Pull request closed without merging**